src/codegen/names.jl

```diff
@@ -5,7 +5,7 @@ const JULIA_RESERVED_KEYWORDS = Set{String}([
     "global", "module", "using", "import", "export", "const", "let",
     "bitstype", "do", "baremodule", "importall", "immutable",
     "Type", "Enum", "Any", "DataType", "Base", "Set", "Method",
-    "Array", "Vector", "Dict", "Union",
```

Comment:
Makes me nervous that there are other keywords we're missing out on here.

Reply:
Thanks for the quick review, Jacob, and happy New Year!

Good point, looking at https://docs.julialang.org/en/v1/base/base/#Keywords I think we are also missing `true` and `false`. I'll add them.

Reply:
Yeah, I think that was the best "source of truth" I could find as well, as there doesn't seem to be anything else in Base itself. Perhaps in JuliaSyntax.jl there's something more authoritative? cc: @c42f